sdk/appconfiguration/app-configuration/test/public/snapshot.spec.ts

```diff
@@ -160,7 +192,7 @@ describe("AppConfigurationClient snapshot", () => {
       const snapshot = await client.getSnapshot(newSnapshot.name);
       assertEqualSnapshot(snapshot, newSnapshot);
 
-      await client.archiveSnapshot(newSnapshot);
+      await client.archiveSnapshot(newSnapshot.name);
     });
 
     it.skip("accepts operation options", async function () {
```

Comment:
why skipped?

Reply:
We open this issue to track skipping of these tests #26447. They are currently failing on the Windows tests

Reply:
Can add a comment referencing the issue, please?

Reply:
The comment was added in the `createSnapshot` test [here](https://github.com/Azure/azure-sdk-for-js/blob/311e5454172030ce63f3534d8128499ff998b1a6/sdk/appconfiguration/app-configuration/test/public/snapshot.spec.ts#L97). I think that should be enough. Do you want to add to all these tests?